docs/UserGuide.md

```diff
@@ -114,18 +114,22 @@ Examples:
 
 Finds persons whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+Format: `find [INDEX] [n/KEYWORDS] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+
+* Searches for the person using the given attribute
+* For search by name,
+  - The name search is case-insensitive. e.g `hans` will match `Hans`
+  - The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+  - Partial words will be matched if they appear at the start of any word in the name. For example:
+    - Searching for `Han` will match `Hans Bo`
+    - Searching for `Bo` will match `Hans Bo`
+    - However, searching for `ans` will not match `Hans Bo`, as `ans` does not appear at the start of a word.
+  - Persons matching at least one keyword will be returned (i.e. `OR` search).
+    - e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
```

Comment:
Maybe we should mention that for find/delete by tag, entries will be only found if all tags match